src/query/service/src/interpreters/access/privilege_access.rs

```diff
@@ -869,6 +869,9 @@ impl AccessChecker for PrivilegeAccess {
             Plan::DropView(plan) => {
                 self.validate_db_access(&plan.catalog, &plan.database, vec![UserPrivilegeType::Drop], plan.if_exists).await?
             }
+            Plan::DescribeView(plan) => {
+                self.validate_table_access(&plan.catalog, &plan.database, &plan.view, vec![UserPrivilegeType::Select], false).await?
```

Comment:
Actually the view is a subquery, so in privilege check, we need to check the table in subquery. If the user has select about these tables, the user can desc the view. cc @flaneur2020 How about you?

Reply:
Or just be consistent for now? Don't do the check. I think it also be reasonable.

Reply:
> If the user has select about these tables, the user can desc the view
+1 it looks make sense to me.

Reply:
So, what we need to do, keep intact or do something？

Reply:
just need add test in this pr. I will modify it in other pr.